recipes/sentry-native/all/conanfile.py

```diff
@@ -16,7 +16,7 @@ class SentryNativeConan(ConanFile):
     topics = ("conan", "breakpad", "crashpad",
               "error-reporting", "crash-reporting")
     exports_sources = ["CMakeLists.txt", "patches/*"]
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
```

Comment:
Is one of these enough?
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```